src/components/AppPagination.vue

```diff
@@ -2,10 +2,8 @@
   <ul class="flex items-center -mx-2">
     <li>
       <button
-        :class="
-          isFirst &&
-          'opacity-25 p-2 bg-primary hover:bg-white hover:font-bold hover:border-link'
-        "
+        class="p-2"
+        :class="isFirst && 'opacity-25 hover:font-bold hover:border-link'"
```

Comment:
Why breaking it down into `class=""` and `:class=""`?

Reply:
Because `p-2` should always be applied and the rest are conditional, I guess the alternative would be
```
:class="isFirst ? 'p-2 opacity-25 hover:font-bold hover:border-link' : 'p-2'"
```